libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RoutingTilesOptions.kt

```diff
@@ -8,14 +10,19 @@ import java.net.URI
  * Routing tiles are stored on-device and used for map-matching (enhanced location production),
  * offline routing, Electronic Horizon generation, and other.
  *
- * @param tilesBaseUri scheme and host, for example `"https://api.mapbox.com"`
+ * @param tilesBaseUri scheme and host, for example `"https://api.mapbox.com"`.
+ * If empty, the navigator works in the fallback mode all the time
+ * (route line following; no full map-matching; no map-matching in free drive).
  * @param tilesDataset string built out of `<account>[.<graph>]` variables.
  * Account can be `mapbox` for default datasets or your username for other.
  * Graph can be left blank if you don't target any custom datasets.
  * @param tilesProfile profile of the dataset.
  * One of (driving|driving-traffic|walking|cycling|truck).
  * @param tilesVersion version of tiles, chosen automatically if empty
- * @param filePath used for storing road network tiles
+ * @param filePath used for persistent configuration and history files storing.
+ * @param tileStore tile store instance. It manages downloads and storage for requests to
+ * tile-related API endpoints. For offline/predictive-caching use cases this instance should be
+ * the same that is passed to map resource options.
```

Comment:
Could you copy these new docs to the builder methods as well? The builder methods are the parts of the API that developers actually interact with.

Reply:
done